src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Colors getRandomColor() {
+        Random random = new Random();
+        int colorIndex = random.nextInt(Colors.values().length);
+
+        return Colors.values()[colorIndex];
```

Comment:
It's more efficient to call Colors.values() once and store it in a local variable, rather than calling it twice (once to get the length and once to access the array). Calling it multiple times causes the array to be cloned each time.